syntax = "proto3";
import "ProtocolClientToServerEnum.proto";
package ProtocolClientToServer;
message RequestLogin
{
	string m_accountID = 1;
}